### DISCUSSION TITLE
problem with images ?

### DISCUSSION THREAD

**Discussion by thib3113** [community/community#128407] [npm]
I have multiple packages with images problems . It seems the images are loaded from github CDN, and not using the url in the readme . 

Here is a recent example : https://www.npmjs.com/package/crowdsec-client . I've re-published, so the image url are alive, but lot of other package has images down : https://www.npmjs.com/package/crowdsec-client-scenarios . 

I also check on a known package axios : https://www.npmjs.com/package/axios . And it seems to be the same problem .

**Comment by taoliujun** [comment]
same problem.
